.swiftlint.yml

```diff
@@ -0,0 +1,110 @@
+# Required Swiftlint Version
```

Comment:
Maybe would be helpful in this file to include a reference to where we pulled it from with a URL link.